helix-term/src/main.rs

```diff
@@ -61,9 +61,9 @@ ARGS:
 FLAGS:
     -h, --help                     Prints help information
     --tutor                        Loads the tutorial
-    --health [LANG]                Checks for potential errors in editor setup
-                                   If given, checks for config errors in language LANG
-    -g, --grammar {{fetch|build}}    Fetches or builds tree-sitter grammars listed in languages.toml
+    --health [CATEGORY]            Checks for potential errors in editor setup
+                                   CATEGORY can be a language or one of 'clipboard' or 'all'
+    -g, --grammar {{fetch|build}}  Fetches or builds tree-sitter grammars listed in languages.toml
```

Comment:
The double space here was intentional actually:

![hxhoffset](https://user-images.githubusercontent.com/21230295/177017721-c1196b03-a69c-4eeb-8c60-95dad0cd0ad4.png)

The double braces become single braces because it's interpreted as an escaped formatting argument so we end up needing those two extra spaces